HelloWorld.java

```diff
@@ -47,4 +47,30 @@ public static void main(String[] args) {
 caclulator_multiplication();
 
 }
+      public class Method {
+ static void GantogtokhMethod(){
+  
+ System.out.println("My name is Gantogtokh Battumur, this is my code");
+ }
+    public static void main(String[] args) {
```

Comment:
Hi, the program already has a main method which is in line 40. If you can, could you please delete this line here and put everything inside the main method? It can be done in line 45. Thank you

Reply:
ok i will do it